### PR TITLE
 [fluentd-elasticsearch] Fix `fluentd-elasticsearch-logs` chart

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Fix `fluentd-elasticsearch-logs`

## why
* `fluentd-elasticsearch-logs` uses new version of `cloudposse-incubator/fluentd-kubernetes` chart with fixes

### CODE REVIEW

File: helmfile.d/0530.fluentd-elasticsearch.yaml [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+
+    rbac:
+      create: {{ env "FLUENTD_ELASTICSEARCH_RBAC_CREATE" | default "false" }}
+
+    serviceAccount:
+      # Specifies whether a ServiceAccount should be created
+      create: {{ env "FLUENTD_ELASTICSEARCH_SERVICE_ACCOUNT_CREATE" | default "true" }}
+      # The name of the ServiceAccount to use.
+      # If not set and create is true, a name is generated using the fullname template
+      name:
```
Comment (osterman) [inline helmfile.d/0530.fluentd-elasticsearch.yaml]: Parameterize

**Review by osterman — CHANGES_REQUESTED** [review]
One small nitpick. Otherwise LGTM

**Review by osterman — DISMISSED** [review]

**Review by osterman — CHANGES_REQUESTED** [review]
Decompose into 2 PRs. One per helmfile.

File: helmfile.d/0520.fluentd-elasticsearch-logs.yaml [hunk 2]
```diff
@@ -48,8 +48,9 @@ releases:
       open:
         FLUENT_ELASTICSEARCH_HOST: '{{ env "ELASTICSEARCH_HOST" }}'
         FLUENT_ELASTICSEARCH_PORT: '{{ env "ELASTICSEARCH_PORT" }}'
+        FLUENT_ELASTICSEARCH_SCHEME: '{{ env "ELASTICSEARCH_SCHEME" }}'
         FLUENT_UID: "0"
-      secret:
-        FLUENT_ELASTICSEARCH_USER: '{{ env "ELASTICSEARCH_USER" }}'
-        FLUENT_ELASTICSEARCH_PASSWORD: '{{ env "ELASTICSEARCH_PASSWORD" }}'
+#      secret:
```
Comment (osterman) [inline helmfile.d/0520.fluentd-elasticsearch-logs.yaml]: Add a conditional rather than comment out. Customer may be using a managed ES in the future.

**Comment by goruha** [comment]
Update
* Fluentd from stable repo - removed
* Reverted user/password env vars

Anything else for merge?

**Review by osterman — APPROVED** [review]

**Pull request merged**